src/SnapshotCodeLens/SnapshotCodeLensProvider.ts

```diff
@@ -20,7 +20,8 @@ export function registerSnapshotCodeLens(enableSnapshotPreviews: boolean) {
 
 class SnapshotCodeLensProvider implements vscode.CodeLensProvider {
   public provideCodeLenses(document: vscode.TextDocument, _token: vscode.CancellationToken) {
-    const snapshots = new Snapshot()
+    // temporarily use `undefined` as parser argument, until Jest/#6212 is published
+    const snapshots = new Snapshot(undefined)

```

Comment:
How do you feel about adding another line to the comment explaining when we can remove the argument? It'd be quick for someone who sees it to test it and see if the type updated type definition with the optional arg has been published with facebook/jest#6212.

Reply:
My plan was to remove it by myself as soon as there is a new release of Jest (I had it written to my calendar), but since jest-editor-supportv23 already includes that change, the `undefined` parameter won't be necessary anymore.